papergrid/src/lib.rs

```diff
@@ -766,12 +778,16 @@ impl Settings {
     }
 
     /// Indent method sets indent for a cell
-    pub fn indent(mut self, left: usize, right: usize, top: usize, bottom: usize) -> Self {
-        self.indent = Some(Indent {
+    pub fn padding(mut self, left: usize, left_char: char, right: usize, right_char: char, top: usize, top_char: char, bottom: usize, bottom_char: char) -> Self {
```

Comment:
I would shorten the argument list by using Indent from my previous comment.
What do you think?

Reply:
I temporary solved it as you say...
But I haven't found a reason to use the "duplicate" of this one in the package `tabled/src/padding.rs`.

Reply:
I also was thinking about it actually.
Maybe the tuple based struct or just a type definition would be OK there.